src/pages/index.ejs

```diff
@@ -200,9 +200,7 @@ description: "Betaflight is flight controller software (firmware) used to fly mu
 						</div>
 						<section class="video-area col-lg-6">
 							<div class="overlay overlay-bg"></div>
-							<div class="container">
-								<div class="video-content">
-			
+								<div class="container">			
```

Comment:
```suggestion
						<div class="container">			
```

We also need to remove line 207

Reply:
Yeah good catch

Reply:
@kensherman  fix this up and we’ll merge it

Reply:
Have run html checker on all files. There's a lot of noise from the esj script tags but think I've got the hmtl ones fixed